doc/wg/network/notes/network-notes-2023-11-02.md

```diff
@@ -0,0 +1,58 @@
+# Tock Network WG Meeting Notes
+
+- **Date:** November 02, 2023
+- **Participants:**
+    - Tyler Potyondy
+    - Branden Ghena
+    - Leon Schuermann
+    - Felix Mada
+- **Agenda**
+    1. Updates
+    2. Buffer Management
+    3. OpenThread Support
+- **References:**
+    - [OpenThread Platforms](https://openthread.io/platforms)
+    - [OpenThread Porting](https://openthread.io/guides/porting)
+
+
+## Updates
+- Leon: Merged STM32 Ethernet PR into Tock-Ethernet! Really high quality code. Next is merging the branch itself
+- Branden: The more you can limit the changes to just "ethernet" stuff, the easier that PR will be to merge
+- Leon: Also needed to increase the stack size for boards. Hoping to fix that. The only change outside of ethernet stuff will be adding this stack to some demo boards
+- Leon: Boards are: STM board now, LiteX simulation, LiteX ARTY board, QEMU with VirtIO, nRF series with ethernet over USB. So we can polish our initial HIL, ensure that things work, then make one coherent PR
+
+
+## Buffer Management
+- Leon: Update is no updates yet! Spent a few hours after the call last time, and almost all of the code compiles, but there's still some that doesn't which may require some architecture redesigns. Might be losing some type information when passing things down that we still need when passing buffers back up.
+- Leon: Even from this current work-in-progress design, I think we can extract a simpler design that will support what we need for some initial tests.
+
+
+## OpenThread Support
+- Tyler: https://openthread.io/guides/porting and https://openthread.io/platforms
+- Tyler: Amit's feedback has been pretty strongly anti-capsule implementation. I think it's made sense to start there. Especially since we've exposed some bugs in 15.4 implementation.
+- Tyler: Longer term though, it needs to be robust and bullet-proof for adoption. So for Thread support, we have a really good opportunity to keep the library as a process in userland.
+- Tyler: There's a guide for OpenThread already to porting and what hardware you need. Requirements are: 15.4 which we have, alarms are good, True RNG is fine on Nordic although other boards are unclear and I don't know if we have an RNG driver, non-volatile storage I'm unsure

```

Comment:
Assuming a small amount of configuration data, the _easiest_ to implement would be something like the UICRs on the nRF chips ( https://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.nrf52832.ps.v1.1%2Fuicr.html ), not sure if Tock has driver support for them yet, but they're super simple.

Other chips might not have such an easy NVM solution, but this would let you avoid flash headaches for the short term.